feature-libs/product-configurator/rulebased/components/price/configurator-price.component.spec.ts

```diff
@@ -138,35 +183,74 @@ describe('ConfiguratorPriceComponent', () => {
         htmlElem,
         '.cx-quantity-price'
       );
-      expect(component.price).toEqual('+ $10');
+      expect(component.price).toEqual('+$10');
     });
 
-    it('should be defined: value price total greater than zero', () => {
+    it('should be defined: a positive total value price is given', () => {
       component.formula = createTestData(0, 0, 150, true);
       fixture.detectChanges();
 
       CommonConfiguratorTestUtilsService.expectElementPresent(
         expect,
         htmlElem,
-        '.cx-price'
+        '.cx-greyed-out'
       );
       CommonConfiguratorTestUtilsService.expectElementNotPresent(
         expect,
         htmlElem,
         '.cx-quantity-price'
       );
-      expect(component.priceTotal).toEqual('+ $150');
+      expect(component.priceTotal).toEqual('+$150');
     });
 
-    it('should be defined: complete price formula', () => {
-      component.formula = createTestData(2, 10, 20);
+    it('should be defined: a negative total value price is given', () => {
+      component.formula = createTestData(0, 0, -150, true);
+      component.formula = {
```

Comment:
You try to define formula twice. Plz, remove the second one

Reply:
done